pay-api/src/pay_api/utils/enums.py

```diff
@@ -355,6 +355,7 @@ class EFTShortnameStatus(Enum):
     LINKED = 'LINKED'
     UNLINKED = 'UNLINKED'
     PENDING = 'PENDING'
+    PENDING_REFUND = 'PENDING_REFUND'
```

Comment:
I mentioned this as well on PR #1719, the EFT Refund flow has it's own flow and states for pending, approval, rejection. There should be a new set of statuses. When we create a refund we are also using the EFTCreditInvoiceStatus enum instead.